server/routes/journeys/referral/involvement/tests.cy.ts

```diff
@@ -0,0 +1,218 @@
+      .should('not.exist')
+    cy.findByRole('link', { name: /enter the staff member’s name/i }).should('not.exist')
+    cy.get('.govuk-error-summary a').should('have.length', 1)
+
+    cy.findByRole('radio', { name: /yes/i }).click()
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.findByRole('textbox', { name: /names of staff assaulted/i }).type('staff stafferson')
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.get('p')
+      .contains(/enter the staff member’s name/i)
+      .should('not.exist')
+    cy.findByRole('link', { name: /enter the staff member’s name/i }).should('not.exist')
+    cy.get('.govuk-error-summary a').should('have.length', 1)
+  }
+
+  const checkProactiveReactiveContent = () => {
+    // We're starting with it being proactive, so test the content and validation messages, then switch to reactive so we can test those as well
+    cy.findByRole('heading', { name: /behaviour involvement/i }).should('be.visible')
+    cy.findByText(/How was Testname User involved in the behaviour\?/).should('be.visible')
+    cy.findByText(/have any staff been assaulted as a result of this behaviour\?/i).should('be.visible')
+
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.findByRole('link', { name: /select how the prisoner has been involved in the behaviour/i }).should('be.visible')
+    cy.findByRole('link', { name: /select if any staff were assaulted as a result of the behaviour or not/i }).should(
+      'be.visible',
+    )
+    cy.get('p')
+      .contains(/select how the prisoner has been involved in the behaviour/i)
+      .should('be.visible')
+    cy.get('p')
+      .contains(/select if any staff were assaulted as a result of the behaviour or not/i)
+      .should('be.visible')
+
+    cy.findByRole('link', { name: /^back/i }).click()
+    cy.findByRole('link', { name: /^back/i }).click()
+
+    cy.findByRole('radio', { name: /reactive/i }).click()
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.findByRole('button', { name: /continue/i }).click()
+
+    cy.findByRole('heading', { name: /incident involvement/i }).should('be.visible')
+    cy.findByText(/How was Testname User involved in the incident\?/).should('be.visible')
+    cy.findByText(/were any staff assaulted during the incident\?/i).should('be.visible')
+
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.findByRole('link', { name: /select how the prisoner was involved in the incident/i }).should('be.visible')
+    cy.findByRole('link', { name: /select if any staff were assaulted during the incident or not/i }).should(
+      'be.visible',
+    )
+    cy.get('p')
+      .contains(/select how the prisoner was involved in the incident/i)
+      .should('be.visible')
+    cy.get('p')
+      .contains(/select if any staff were assaulted during the incident or not/i)
+      .should('be.visible')
+  }
+
+  const setupDataSignInAndGo = () => {
```

Comment:
If you visit the start page for the journey, inject your test data and then manually visit the page under test we shouldn't need to repeat ourselves with all of this csip mock data. See the record-decision/confirmation tests

Reply:
@JackReeveMoJ so if I did that, I'd be able to remove `csipRecord` entirely?

Reply:
Yep, because it should be covered in the initial start page when that grabs the csip mock

Reply:
That's done now :)